packages/ignite-ir-boilerplate-2016/boilerplate.js

```diff
@@ -1,6 +1,21 @@
+const Shell = require('shelljs')
 const options = require('./options')
 const { merge, pipe, assoc, omit, __ } = require('ramda')
 
+const verifyAndroidInstalled = function (context) {
+  let androidInstalled = true
+  if (!Shell.which('android')) {
+    console.log(`Unable to find 'android' in your PATH.`)
+    androidInstalled = false
+  }
+  const androidPath = process.env['ANDROID_HOME']
+  if (!androidPath || !Shell.test('-d', androidPath)) {
```

Comment:
I'll be able to use `system.exists` here if https://github.com/infinitered/gluegun/pull/159 is pulled in.